fiddler/manifest.json

```diff
@@ -0,0 +1,84 @@
+{
+  "manifest_version": "2.0.0",
+  "app_uuid": "ee617671-508e-4bb3-ba25-8815b11a16aa",
+  "app_id": "fiddler",
+  "display_on_public_website": true,
+  "tile": {
+    "overview": "README.md#Overview",
+    "configuration": "README.md#Setup",
+    "support": "README.md#Support",
+    "changelog": "CHANGELOG.md",
+    "description": "Fiddler provides visibility and explainability for AI model performance",
```

Comment:
The description should be 1-2 sentences on the value of the dashboard for customers